src/components/form/time-picker/time-picker.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from "react";
+import {
+  LocalizationProvider,
+  TimePicker,
+  TimeStepOptions,
+  TimeView,
+} from "@mui/x-date-pickers";
+import {
+  Controller,
+  ControllerProps,
+  FieldPath,
+  FieldValues,
+} from "react-hook-form";
+import { ForwardedRef, forwardRef } from "react";
+import { AdapterDateFns } from "@mui/x-date-pickers/AdapterDateFns";
+import useLanguage from "@/services/i18n/use-language";
+import en from "date-fns/locale/en-US";
+
+export const getValueByKey = (language: string) => {
+  switch (language) {
+    case "en":
+      return en;
+    default:
+      return en;
+  }
+};
```

Comment:
Let's create in `src/components/form` new folder `date-pickers`. Move `date-picker.tsx` and `time-picker.tsx` to `date-pickers` folders. Also please create in this folder `helper.ts` and move `getValueByKey` to this file.